README.md

```diff
@@ -0,0 +1,37 @@
+# Crane Cloud Documentation
+> A centralised location for all information regarding usage of the platform.
+
+## How to setup the project:
+ #### Clone the repository
+  ` git clone https://github.com/crane-cloud/documentation.git`
+
+ * If `develop` is not up to date, `git checkout develop` for it is the default base branch for this project.
+#### Install pip or upgrade its version by running
+ 
+     > `python3 -m pip install --upgrade pip`
+
+#### Install and creating a virtual environment run
+ 
+     > `python3 -m pip install virtualenv`
+
+#### To create a virtual environment run
+
+     > `python3 -m virtualenv name-of-your-virtual-environment`
+
+**NOTE** Remeber to add the virtual environment to your gitignore.
```

Comment:
@neelxie Just a typo to correct *Remember

Reply:
Yeah apart from this it looks good to me

Reply:
@MutegekiHenry @Mubangizi this has been fixed.

Reply:
looks good